### DISCUSSION TITLE
plugin-diagram Not Working for Mindmap Diagrams

### DISCUSSION THREAD

**Discussion by TessaJRD** [Milkdown/milkdown#1490] [Q&A]
I'm experiencing an issue with the plugin-diagram in my Milkdown project that I have set up using Vue. Specifically, the plugin does not seem to render mindmap diagrams correctly. Other types of diagrams, such as flowcharts, are working without any problems. When I insert a flowchart diagram, it renders correctly. (as in the picture)
![flowchart diagram](https://github.com/user-attachments/assets/214ca18d-4db6-41cc-bf2f-10d92f34bbba)
When I try to create a mindmap diagram, it does not render at all. 
![mindmap diagram](https://github.com/user-attachments/assets/878c3004-f053-44a0-99df-24b3cd056f4b)
I see syntax error as result. If I add a newline within the diagram syntax, it renders, but all the elements are overlapping.
![newline within the diagram syntax](https://github.com/user-attachments/assets/d7fa18b8-46d0-4fff-9ac7-37d29f0bbd6a) 
I also see an error message in the console:
![error message](https://github.com/user-attachments/assets/d5e10971-006f-4b1e-818d-8fa99af3af7d)

Could someone please provide guidance on how to get mindmap diagrams working with the plugin-diagram in a Milkdown project using Vue? Are there any additional configurations or dependencies required?

**Comment by johnjazzi** [comment]
Can you share the code for the editor config?
